packages/expo-document-picker/ios/EXDocumentPicker/EXDocumentPickerModule.m

```diff
@@ -111,13 +119,19 @@ - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
     @try {
       // TODO: drop #if macro once Xcode is updated to 12
 #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 140000
-      if (@available(iOS 14, *)) {
-        UTType* utType = EXConvertMimeTypeToUTType(mimeType);
-        documentPickerVC = [[UIDocumentPickerViewController alloc] initForOpeningContentTypes:@[utType] asCopy:YES];
+      if ((@available(iOS 14, *))) {
+        NSMutableArray *uttArray = [mimeTypes mutableCopy];
```

Comment:
We prefer verbosity 👍 
It's always good to omit almost all abbreviations in open source software, or even in software altogether 😉 
```suggestion
        NSMutableArray *utTypes = [mimeTypes mutableCopy];
```